manual/config/seafile-conf.md

```diff
@@ -205,6 +205,13 @@ ca_path = /etc/mysql/ca.pem
 ```
 When set `use_ssl` to true and `skip_verify` to false, it will check whether the MySQL server certificate is legal through the CA configured in `ca_path`. The `ca_path` is a trusted CA certificate path for signing MySQL server certificates. When `skip_verify` is true, there is no need to add the `ca_path` option. The MySQL server certificate won't be verified at this time.
 
+Since Seafile 10.0.3, if your mysql service supports the unix_socket authentication plugin, then if you specify the option of unix_socket and do not specify the option of user, the unix socket will be used for authentication by default.
```

Comment:
Since Seafile 10.0.3 and 10.0.7 Pro, you can use [unix_socket authentication plugin](https://mariadb.com/kb/en/authentication-plugin-unix-socke) provided by MariaDB/MySQL. To enable it, you need to specify the unix_socket option without user name and password.